documentation/src/docs/metrics.md

```diff
@@ -121,7 +121,7 @@ This is an example from io.pravega.segmentstore.server.host.handler.PravegaReque
 public class PravegaRequestProcessor extends FailingRequestProcessor implements RequestProcessor {
```

Comment:
Not related to your PR, but why in the world do we have a (now obsolete) copy of this class in `metrics.md` ?